### PR TITLE
add actions for backup and restart

### PR DESCRIPTION
**Pull request by rdlucas2**

### CODE REVIEW

File: .github/workflows/backup.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        with:
+          aws-region: ${{ secrets.AWS_REGION }}
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          instance-ids: ${{ secrets.INSTANCE_ID }}
+
+          #TODO: update these values to point to the backup script
+          working-directory: /home/ubuntu/application
+          command: ls -al #something like -> /bin/sh backup.sh
+          comment: Hello world!
```
Comment (rcatlin) [inline .github/workflows/backup.yml:25]: Change to:
```
working-directory: /home/ec2-user/
command: ./stop_backup_start.sh
comment: StopBackupStart
```

The script we're using is here: https://github.com/rcatlin/valheim-server-cdk/blob/master/templates/stop_backup_start.sh.hbs

File: .github/workflows/update.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+name: AWS SSM Send-Command Update
```
Comment (rcatlin) [inline .github/workflows/update.yml:1]: Don't need this as of yet.

File: README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
-# Valhdiem Server with CDK
+# Valhiem Server with CDK
 
-An "easy" way to setup the infrastructure and a running valheim server (with manual backup capabilities) on AWS!
+An "easy" way to setup the infrastructure and a running Valheim server (with manual backup capabilities) on AWS!
```
Comment (rcatlin) [inline README.md:3]: Let's leave these out of the PR.

I changed the header spelling so please rebase. :+1:

**Review by rcatlin — CHANGES_REQUESTED** [review]
Small comments

**Comment by rcatlin** [comment]
Awesome PR, thank you! Just a few small comments and we can get this merged. Excited to get this running.

**Comment by rcatlin** [comment]
Also, please squash into 1️⃣  commit 

🦑

**Comment by rdlucas2** [comment]
Going to update and condense the changes in a new branch.

**Pull request closed without merging**